spec/system/avo/test_helpers_spec.rb

```diff
@@ -222,6 +222,14 @@
         expect(remove_tag(field: :tags, tag: "one")).to eq ["three"]
         expect(tag_suggestions(field: :tags, input: "")).to eq ["one", "two"]
       end
+
+      let!(:post) { create :post, tag_list: ["test", "five", "seven"] }
+
+      it "verify tags" do
+        visit avo.edit_resources_post_path(post)
+
+        expect(tags(field: :tags)).to eq ["test", "five", "seven"]
```

Comment:
It looks like in the test this is coming out as `[“seven”, “five”, “test”]`

Reply:
True, it was passing locally, but the tags field on the post has enforced suggestions so it's flaky... Just changed it to `["one", "two"]` should work as expected not